jupyterlab_comments/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "jupyterlab-comments",
+  "version": "0.1.0",
+  "description": "Collaborative notebooks in git",
+  "keywords": [
+    "jupyter",
+    "jupyterlab",
+    "jupyterlab-extension"
+  ],
+  "homepage": "https://github.com/my_name/myextension",
+  "bugs": {
+    "url": "https://github.com/my_name/myextension/issues"
+  },
+  "license": "BSD-3-Clause",
+  "author": "mkalil",
+  "files": [
+    "lib/**/*.{d.ts,eot,gif,html,jpg,js,js.map,json,png,svg,woff2,ttf}",
+    "style/**/*.{css,eot,gif,html,jpg,json,png,svg,woff2,ttf}"
+  ],
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "style": "style/index.css",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/my_name/myextension.git"
+  },
+  "scripts": { 
+    "build": "tsc",
+    "clean": "rimraf lib tsconfig.tsbuildinfo", 
+    "eslint": "eslint . --ext .ts,.tsx --fix",
+    "eslint:check": "eslint . --ext .ts,.tsx",
+    "prepare": "jlpm run clean && jlpm run build",
+    "watch": "tsc -w"
+  },
+  "dependencies": {
+    "@jupyterlab/application": "^2.0.0"
```

Comment:
AI Platform Notebooks uses JupyterLab 1.2.x and that's what we've targeted for the other extensions in this repository. 

@ojarjur - This might be something we need to discuss with the team because the last I was told, there are no immediate plans to move to 2.0.x.